RF24Mesh.cpp

```diff
@@ -154,7 +154,24 @@ void ESBMesh<network_t, radio_t>::setChild(bool allow)
 template<class network_t, class radio_t>
 bool ESBMesh<network_t, radio_t>::checkConnection()
 {
-    // getAddress() doesn't use auto-ack; do a double-check to manually retry 1 more time
+
+    if (!_nodeID) return false;
+    if (mesh_address == MESH_DEFAULT_ADDRESS) return false;
+
+// Connection check via parent node
+#if RF24MESH_CONN_CHECK_TYPE == RF24MESH_CONN_CHECK_PARENT
+    RF24NetworkHeader header;
+    header.to_node = network.parent();
+    header.type = NETWORK_PING;
+    for (uint8_t i = 0; i < MESH_CONNECTION_CHECK_ATTEMPTS; i++) {
```

Comment:
Using `++i` is more performant than `i++`.

I missed this last time we changed this function's behavior, so the `for` loop below (in old behavior) could use this advice as well.

Reply:
resolved in https://github.com/nRF24/RF24Mesh/pull/250/commits/4b1e3a2895f35db40284158248669d22a98d2781